style.css

```diff
@@ -101,3 +103,354 @@
 .social-media-icons .mail {
   height: 20px;
 }
+
+#work {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto auto auto;
+  padding: 114px 24px;
+  gap: 90px;
+  background: #e5e5e5;
+}
+
+.card-work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [ ] [REQUIRED] Please make changes!

- 1-Your list of project technologies is a bit large and the font thickness is not like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
- 2-There is a bit of a problem with your work section cards because the layout of the cards changes as you expand and shrink the project from 300 px to 700 px.

![image](https://user-images.githubusercontent.com/63915024/170841889-4fadb724-1226-49d8-8bdf-0ff56f4d983e.png)

Reply:
Thanks for the review.